shared/src/business/useCases/getConsolidatedCaseDeadlinesInteractor.ts

```diff
@@ -0,0 +1,41 @@
+import { UnknownAuthUser } from '@shared/business/entities/authUser/AuthUser';
+import { UnauthorizedError } from '@web-api/errors/errors';
+import {
+  isAuthorized,
+  ROLE_PERMISSIONS,
+} from '@shared/authorization/authorizationClientService';
+import { getCaseDeadlinesByConsolidatedCaseDeadlineId } from '@web-api/persistence/postgres/caseDeadlines/getCaseDeadlinesByConsolidatedCaseDeadlineId';
+import { getCaseByDocketNumberPostgres } from '@web-api/persistence/postgres/cases/getCaseByDocketNumber';
+
+export async function getConsolidatedCaseDeadlinesInteractor(
+  _applicationContext: IApplicationContext,
+  {
+    consolidatedCaseDeadlineId,
+  }: {
+    consolidatedCaseDeadlineId: string;
+  },
+  authorizedUser: UnknownAuthUser,
+): Promise<{ docketNumber: string; caseCaption: string }[]> {
+  if (!isAuthorized(authorizedUser, ROLE_PERMISSIONS.CASE_DEADLINE)) {
+    throw new UnauthorizedError('Unauthorized');
+  }
+
+  const DEADLINES = await getCaseDeadlinesByConsolidatedCaseDeadlineId(
+    consolidatedCaseDeadlineId,
+  );
+
+  const RESULTS: { docketNumber: string; caseCaption: string }[] = [];
+  for (let index = 0; index < DEADLINES.length; index++) {
+    const { docketNumber } = DEADLINES[index];
+    const [CASE] = await getCaseByDocketNumberPostgres(docketNumber);
```

Comment:
We should never be calling this outside of `getCaseByDocketNumber`. It is a temporary function (deleted in 10502). Instead, we should call `getCaseMetadataByDocketNumber`. This is currently in dynamo, but it is in Postgres in 10502/ustc-test.

I should have put a comment to that effect in the file. I apologize.